.gitignore

```diff
@@ -102,3 +103,7 @@ target/
 
 # Hidden files
 .*
+
+#plugins
+plenum/server/plugin/*
```

Comment:
We have some pre-defined plugins in `plenum/server/plugin`. Why add it to gitignore?

Reply:
Because the plugin no longer looks into this directory but now looks for dependencies

Reply:
But plenum already has code there, and may have it in future.

Reply:
Sorry, that was temporary, will change it